tools/test-ci.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd `dirname $0`/.. > /dev/null
+if [ -z ${PYTHON+x} ]; then export PYTHON=`which python`; fi
```

Comment:
```PYTHON=`which python` ``` is for me, when running in WSL (Windows System for Linux) there's a bad behaviour where it'll try to run the windows `python` over the Ubuntu one 🤷‍♀️